pkg/build/iso.go

```diff
@@ -146,3 +153,23 @@ func (b *Builder) createIsoCommand(logFilename, scriptName string) (*exec.Cmd, *
 
 	return cmd, logFile, nil
 }
+
+func (b *Builder) findExtractedRawImage() (string, error) {
+	var foundFile string
+	f := func(s string, info os.FileInfo, e error) error {
+		if e != nil {
+			return e
+		}
+		if strings.HasSuffix(info.Name(), ".raw") {
+			foundFile = info.Name()
+		}
+		return nil
+	}
+
+	rawExtractPath := filepath.Join(b.context.BuildDir, rawExtractDir)
+	if err := filepath.Walk(rawExtractPath, f); err != nil {
+		return "", fmt.Errorf("traversing raw extract directory %s: %w", rawExtractPath, err)
+	}
+
+	return filepath.Join(rawExtractPath, foundFile), nil
```

Comment:
What if `foundFile` is still an empty string here?

Reply:
Good pickup.